.ansible-lint

```diff
@@ -0,0 +1,3 @@
+skip_list:
```

Comment:
Instead of this file please add ` -x 701` to the ansible lint run

Reply:
Done.
Thanks